tests/js/spec/views/acceptProjectTransfer.spec.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import {mount} from 'enzyme';
+import AcceptProjectTransfer from 'app/views/acceptProjectTransfer';
+
+jest.mock('jquery');
+
+describe('AcceptProjectTransfer', function() {
+  beforeEach(function() {
+    MockApiClient.clearMockResponses();
+
+    MockApiClient.addMockResponse({
+      url: '/accept-transfer/',
+      method: 'GET',
+      body: {
+        project: TestStubs.Project(),
+        organizations: [TestStubs.Organization({teams: [TestStubs.Team()]})],
+      },
+    });
+  });
+
+  it('renders', function() {
+    let wrapper = mount(
+      <AcceptProjectTransfer location={{}} />,
+      TestStubs.routerContext()
+    );
+
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Instead of a snapshot can we maybe test
1) that the accept transfer details API endpoint gets called
2) the form can submit and POST to endpoint?